components/04-reference-pages/08-news-page/news-page--sidebar.hbs

```diff
@@ -77,7 +77,7 @@
           {{ render '@blockquote--inline' }}
           <div class="news-text-block">
             <figure class="figure float-start me-3">
-              <img src="/college-dls/college/images/News_Blog.png" class="figure-img img-fluid" alt="Two students speaking at a table" />
+              <img src="{{ image }}" class="figure-img img-fluid" alt="Two students speaking at a table" />
```

Comment:
think this one needs to be `{{path image}}` to render correctly

Reply:
Updated!